### PR TITLE
Add to posh policy

### PR DESCRIPTION
**Pull request by vidya-ram**

### CODE REVIEW

File: funnel/views/index.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ class PolicyPage:
     PolicyPage('policy/refunds', __("Cancellation &amp; refund policy")),
     PolicyPage('policy/community', __("Community guidelines")),
     PolicyPage('policy/code', __("Code of conduct")),
+    PolicyPage('policy/posh', __("Prevention of Sexual Harassment Policy")),
```
Comment (jace) [inline funnel/views/index.py:41]: This should be in sentence case like the others. It _may_ also need to be abbreviated to "POSH policy" to fit the sidebar.

File: funnel/pages/about/policy/posh.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+**Hasgeek's position on speech and expression**:
+
+All conversations in the groups monitored by Hasgeek i.e., where a Hasgeek employee is the admin, on messaging platforms such as Telegram, WhatsApp and others, will adhere to moderation guidelines established for the groups. Physical meetings conducted through Hasgeek are covered by the Chatham House Rule. This is a Rule where participants are free to use the information received, but should not reveal the identity or the affiliation of the speaker or other participant.
+
+This ensures freedom of speech and facilitates open discussions and dialogue. No acts of abuse/harassment/discrimination/sexual harassment will be condoned or excused. Appropriate action will be taken against the Respondent as per this Policy if the alleged act of harassment has a sexual intent.
+
+Hasgeek cannot be held responsible or liable for the activity of any person below the age of 18. Parents are requested to monitor their children's online activities and ensure their safety. The [community guidelines](https://hasgeek.com/about/policy/community) also state that the age limit of 18 years or older is required to sign up with Hasgeek.
```
Comment (jace) [inline funnel/pages/about/policy/posh.md:257]: "are" seems to be broken up into "a re" here.

**Review by jace — COMMENTED** [review]
The text may need a round of proof reading. The rest looks okay.

File: funnel/pages/about/policy/posh.md
Comment (anweshasen25) [inline funnel/pages/about/policy/posh.md:1]: "her/him" to be changed to "they"
"he/she/they chooses" to be changed to "they choose"

Rest looks fine.

**Pull request merged**